src/main/java/bookmarked/command/FindCommand.java

```diff
@@ -11,8 +11,10 @@
 public class FindCommand extends Command {
     public static final int FIND_KEYWORD_START_INDEX = 5;
```

Comment:
Nice to declare variable names for constants to make code more readable and understandable